### ISSUE TITLE
Windows detecting "No Internet" 

### ISSUE THREAD

**Issue by RichardTHF** [vernesong/OpenClash#1216]
I am having a really weird issue where Windows 10 is detecting no internet, usually this wouldn't bother me but in turn it is causing issues with Outlook also not detecting any internet yet the internet is very obviously working. 

![image](https://user-images.githubusercontent.com/5789676/107840443-af42cf00-6ded-11eb-94e7-615608962f79.png)

**Comment by RichardTHF** [comment]
Ok , so done a bit more looking into this and it's getting really weird.... I formatted my comptuer, totally fresh install of Windows 10 and the error seems to come to fruition after I install `February 9, 2021—KB4601319 (OS Builds 19041.804 and 19042.804)` ([Link](https://support.microsoft.com/en-us/topic/february-9-2021-kb4601319-os-builds-19041-804-and-19042-804-87fc8417-4a81-0ebb-5baa-40cfab2fbfde)) or February 9, 2021-KB4601050 Cumulative Update for .NET Framework 3.5 and 4.8 for Windows 10, version 2004, Windows Server, version 2004, Windows 10, version 20H2, and Windows Server, version 20H2 ([Link](https://support.microsoft.com/en-us/topic/february-9-2021-kb4601050-cumulative-update-for-net-framework-3-5-and-4-8-for-windows-10-version-2004-windows-server-version-2004-windows-10-version-20h2-and-windows-server-version-20h2-7d61f856-72ce-7f4c-7e43-76f8aadf2aca)) but neither talk about Proxy settings anywhere.

I can fix Windows saying "No Internet" by going to Settings -> Network & Security -> Proxy and manually entering the proxy settings (192.168.1.1 & Port: 7890) but then when it comes to installing specific bits of software i.e. Microsoft Office i get an error like this 

![image](https://user-images.githubusercontent.com/5789676/107841575-deaa0980-6df6-11eb-8423-261d5c53c41d.png)

If I then turn the Proxy settings to off, the icon goes back to "No Internet" once I have rebooted Windows, but I am able to install Office 365. 

![image](https://user-images.githubusercontent.com/5789676/107841579-f1244300-6df6-11eb-9940-a9af0ff86ffe.png)

I am 99% sure it is not a configuration issue on Clash or OpenClash on my end, but may be wrong about that. It works perfectly with my other devices, both Mac and PC, and the other PC has not done the latest Windows Update.
